samples/bluetooth/mesh/light_switch/child_image/hci_rpmsg.conf

```diff
@@ -5,3 +5,4 @@
 #
 
 CONFIG_BT_EXT_ADV=y
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
```

Comment:
Should it be 1 for LPN?

Reply:
it is not a problem to have an additional adv sets in net core unused.